recipes/pandemy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "Pandemy" %}
+{% set version = "1.0.0" %}
```

Comment:
I noticed that version 1.1.0 is already available. Do you really want to publish 1.0.0 as conda package? Or should you bump to 1.1.0 this recipe?

Reply:
Hi beenje!

Yes I want to publish version 1.0.0 before version 1.1.0. I then want to go through the process of releasing a new version of the recipe (version 1.1.0) as practice exercise.